docs/tidb-binlog/zh_CN/tidb-binlog-issue.md

```diff
@@ -0,0 +1,90 @@
+---
+title: tidb-binlog 问题反馈及排查流程
+category: tools
+---
+
+# 问题反馈及排查流程
+
+## 检查 TiDB-Binlog 运行情况
+
+在上报问题前，请根据 checklist 列出的事项对 binlog 的运行情况进行检查。
+
+### kafka 版本
+* TiDB 是否开启了 Binlog
+    * 查看 TiDB 启动脚本及配置，或者直接查看 TiDB 的日志中记录的配置参数，开启 binlog 的命令行参数为 binlog-socket，在配置文件中为 [binlog] 中的 socket。
+* 查看 Pump、Drainer 的监控
+    * 是否有指标异常。
```

Comment:
we should add docuement how to determine the metrics anomaly， please mark it in jira issue